rest/test-credentials/test-sms-messages-example-1/test-sms-messages-example-1.4.x.php

```diff
@@ -7,5 +8,10 @@
 $token = "your_auth_token"; 
```

Comment:
Should we have some naming convention for the _account SID_ and the _authentication token_?
It will be more consistent with the other examples if we use something like:

```php
$accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"; 
$authToken = "your_auth_token"; 
$client = new Services_Twilio($accountSid, $authToken);
```

What do you think?